Gemfile

```diff
@@ -34,13 +34,14 @@ gem 'webpacker', '~> 4.0'
 # For configuration
 gem 'config', '~> 1.7'
 gem 'honeybadger'
+gem 'nokogiri'
```

Comment:
does it even need its own require?  won't hold up PR, but if true, a followon would be good.

Reply:
Removed!